site/src/modules/resources/AgentRow.tsx

```diff
@@ -267,6 +281,22 @@ export const AgentRow: FC<AgentRowProps> = ({
 					</section>
 				)}
 
+				{containers && (
+					<section>
+						{containers.map((container) => {
+							return (
+								<AgentDevcontainerCard
+									key={container.id}
+									container={container}
+									workspace={workspace}
+									host={proxy.preferredWildcardHostname || window.location.host}
+									agentName={agent.name}
+								/>
+							);
+						})}
+					</section>
```

Comment:
Should we skip showing a card for a stopped devcontainer? Or continue showing it but indicate it's stopped?